src/gsIO/gsWriteParaview.hpp

```diff
@@ -489,6 +489,12 @@ void writeSingleGeometry(gsFunction<T> const& func,
 
         if ( n == 1 )
         {
+            if (d==3)
```

Comment:
This one I do not understand.

Reply:
In here we have n==1, so we plot a scalar function on its domain.
When the domain is less than 3, the graph is shown in 3D.
The new if is for a scalar function on a 3D domain: the scalar field will be shown as color over the 3D domain.

Reply:
Thanks